manifests/base/code-server-notebook-imagestream.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         opendatahub.io/notebook-python-dependencies: '[{"name":"code-sever","version":"4.16"}]'
         openshift.io/imported-from: quay.io/opendatahub/workbench-images
         opendatahub.io/workbench-image-recommended: 'true'
+        opendatahub.io/notebook-build-commit: "1c5bcbfc222d"
```

Comment:
Moreover, I would suggest to use the short git sha version, 7digits. 
ex: 
![image](https://github.com/opendatahub-io/notebooks/assets/42587738/0b82a2f1-303b-4f6b-bd60-89c328bcb1a3)


This will help us later on to use  [ref. link]( https://github.com/opendatahub-io/notebooks/blob/main/.github/workflows/notebooks-digest-updater-upstream.yaml#L65 )